charts/trento-server/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: trento/trento-server:1.0.3
-#!BuildTag: trento/trento-server:1.0.3-build%RELEASE%
+#!BuildTag: trento/trento-server:1.0.4
+#!BuildTag: trento/trento-server:1.0.4-build%RELEASE%
 apiVersion: v2
 name: trento-server
 description: The trento server chart contains all the components necessary to run a Trento server.
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 1.0.3
+version: 1.0.4
```

Comment:
Updating this file and running a `helm dependency update charts/trento-server` didn't update the `Chart.lock`.
I had to remove it and run the update again :thinking:
Am I doing it correctly? :sweat_smile: 

I also changed the `#!BuildTag: bla bla` section in this file myself, is it how it should be or should it happen automatically?

Also other question: do we need to also update the `version` in `charts/trento-server/charts/trento-web/Chart.yaml`

Reply:
1) not sure, should have worked the first time
2) yes